### DISCUSSION TITLE
Needed Options

### DISCUSSION THREAD

**Discussion by fanniepro** [Shabinder/SpotiFlyer#182] [Ideas]
Hi

The app is great hey😀, enjoying it but there are some option which would make it greater,  1. album art quality, 2. the naming convention allow choosing how the album tracks are named, and I have some more ideas but I know its still early stages for the app, but I will learn Kotlin just for this project cause it is very interesting. 

Keep up the good work.👍👍

**Comment by Shabinder** [comment]
Hey @fanniepro ,
I have received many requests for similar features and I am working on them and many more changes , 
for **latest dev** checkout https://github.com/Shabinder/SpotiFlyer/tree/better_error_handling 

and thanks for your kind words. :smile:
